createbillboardplaylist.py

```diff
@@ -311,6 +311,14 @@ def create_all(self):
             "http://www.billboard.com/charts/hot-100",
         )
 
+		# Billboard Country 50
+        self.create_playlist_from_chart(
+            "country-50",
+            "Country 50",
+            "top 50",
```

Comment:
This string needs to have a space character at the end, since it's interpolated into the description right before the next word.

```suggestion
            "top 50 ",
```